fv3core/stencils/moist_cv.py

```diff
@@ -343,7 +342,11 @@ def compute_last_step(
     gz: FloatField,
 ):
     grid = spec.grid
-    moist_pt_last_step(
+
+    # Temporary Fix for calling moist_pt_last_step for verification tests
+    last_step = gtstencil(moist_pt_last_step)
```

Comment:
if this is just for the test code, a FrozenStencil can be made in the translate class. But I am revisiting this particular file and how tests run code from it, so if you prefer to leave it I can incorporate that into the other story.

Reply:
Thanks for the heads up!  I tried adding the FrozenStencil into `translate_last_step.py` and created __call__, but the test code looked for a `compute` method.  I'll wait later to revise this.